crates/prof/src/aggregate.rs

```diff
@@ -172,8 +179,63 @@ impl GroupedMetrics {
             ..Default::default()
         };
         metrics.compute_total();
+        metrics.bounded_par_by_group = self
+            .compute_bounded_par_times()
+            .into_iter()
+            .map(|(k, v)| (k, MdTableCell::new(v, Some(0.0))))
+            .collect();
+
         metrics
     }
+
+    /// Compute per-group parallel proof time with bounded parallelism (NUM_PARALLEL_PROOFS slots).
+    fn compute_bounded_par_times(&self) -> HashMap<String, f64> {
+        let mut per_group = HashMap::new();
+
+        for (group_name, metrics) in &self.by_group {
+            if group_name.contains("keygen") {
+                continue;
+            }
+
+            let mut group_time = 0.0;
+
+            // Add serial execution time for app_proof
+            if group_name == "app_proof" {
+                group_time += sum_metric_ms(metrics, EXECUTE_METERED_TIME_LABEL);
```

Comment:
is this just the time for this one segment?

what we should add here is sum of metered execution time up to (and excluding) this segment -- and no need to count both metered and E1 since we only need metered

Reply:
we don't have the breakdown of metered execution per segment. but i did add avg e1 instead of total e1 like we do in total parallel proof time